modules/orca/main.tf

```diff
@@ -86,6 +87,8 @@ module "orca_lambdas_secondary" {
   orca_reconciliation_lambda_memory_size = var.orca_reconciliation_lambda_memory_size
   orca_reconciliation_lambda_timeout     = var.orca_reconciliation_lambda_timeout
   s3_report_frequency                    = var.s3_report_frequency
+  powertools_service_name                = var.powertools_service_name
```

Comment:
Is the `powertools_service_name` needed? I do not see where `var.powertools_service_name` is defined or needed. In the Lambda we are hard coding this value anyway based on service type.

Reply:
oops forgot to remove this. Removed